bigbluebutton-html5/imports/api/presentations/server/handlers/presentationExport.js

```diff
@@ -1,16 +1,22 @@
 import { check } from 'meteor/check';
 import sendExportedPresentationChatMsg from '/imports/api/presentations/server/handlers/sendExportedPresentationChatMsg';
 import setPresentationExporting from '/imports/api/presentations/server/modifiers/setPresentationExporting';
+import setOriginalUriDownload from '/imports/api/presentations/server/modifiers/setOriginalUriDownload';
 
 export default async function handlePresentationExport({ body }, meetingId) {
   check(body, Object);
   check(meetingId, String);
 
-  const { fileURI, presId } = body;
+  const { fileURI, presId, typeOfExport } = body;
 
   check(fileURI, String);
   check(presId, String);
+  check(typeOfExport, Match.Maybe(String));
 
-  await sendExportedPresentationChatMsg(meetingId, presId, fileURI);
+  if (typeOfExport === "Original") {
+    await setOriginalUriDownload(meetingId, presId, fileURI)
```

Comment:
```suggestion
  if (typeOfExport === 'Original') {
    await setOriginalUriDownload(meetingId, presId, fileURI);
```

Consider enabling your IDE to show lint errors!